Documentation/kubelet-wrapper.md

```diff
@@ -27,9 +27,13 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
 
 In the example above we set the `KUBELET_VERSION` and the kubelet-wrapper script takes care of running the correct container image with our desired API server address and manifest location.
 
-### Customizing rkt Options
+## Customizing rkt Options
 
-Passing customized options or flags to rkt can be accomplished with the RKT_OPTS environment variable. Referencing it in a unit file is straightforward. Example below mounts host's `/etc/resolv.conf` file directly into the container:
+Passing customized options or flags to rkt can be accomplished with the RKT_OPTS environment variable. Referencing it in a unit file is straightforward.
+
+### Use the host's DNS configuration
```

Comment:
When these are H3s, we can link directly to them from the sidebar on the website